test/runBTagAnalyzer_cfg.py

```diff
@@ -50,12 +50,12 @@
     VarParsing.varType.bool,
     "Use Puppi candidates for b tagging"
 )
-options.register('mcGlobalTag', 'FIXME',
+options.register('mcGlobalTag', '106X_mc2018_realistic_v11_L1v1',
```

Comment:
I don't think this fix is needed, because in principle this parameter is taken from the "default" sets, but It doesn't hurt either so you can leave it in I guess ;-)

Reply:
the MC GT is already set as seen from https://github.com/soureek/RecoBTag-PerformanceMeasurements/blob/UL18PreliminaryJECs/test/runBTagAnalyzer_cfg.py#L53
However, I changed the data GT in https://github.com/soureek/RecoBTag-PerformanceMeasurements/blob/UL18PreliminaryJECs/test/runBTagAnalyzer_cfg.py#L58 
to v28 as indicated in https://docs.google.com/presentation/d/1gEF2jExqS1hWs-dlYgxrTT2xQztYbBZcrLdw45Aiy7g/edit#slide=id.g80bfa3ac7d_0_8